pandas/core/series.py

```diff
@@ -1066,7 +1066,9 @@ def _get_value(self, label, takeable: bool = False):
 
             new_index = mi[loc]
             new_index = maybe_droplevels(new_index, label)
-            new_ser = self._constructor(new_values, index=new_index, name=self.name)
+            new_ser = self._constructor(
+                new_values, index=new_index, name=self.name, copy=False
+            )
```

Comment:
Same here: are we sure this is a copy?

Reply:
Did you already have another PR for this?

Reply:
https://github.com/pandas-dev/pandas/pull/51950/files